api/pkg/cli/cmd/install/install.go

```diff
@@ -134,13 +136,35 @@ func (opts *options) run() error {
 		}
 	}
 
+	kube, err := opts.cs.KubeClient()
+	if err != nil {
+		return err
+	}
+
+	version, err := tknVer.GetPipelineVersion(&cli.Clients{Kube: kube}, "")
+	minVersion, err := opts.hubRes.MinPipelinesVersion()
+	if err != nil {
+		return err
+	}
+
+	out := opts.cli.Stream().Out
+
+	minVersion = "v" + minVersion
+
+	if version == "" {
+		printer.New(out).String("WARN: tekton pipelines version unknown, this resource is compatible with pipelines min version " + minVersion)
```

Comment:
should we warn before installing?

Reply:
what if we install it and if it fails then along with error msg add this?

Reply:
> should we warn before installing?

this is the case when Pipelines version is unknown but Pipelines is already installed on the cluster.

> what if we install it and if it fails then along with error msg add this?

The task would fail at the runtime and the failure reason can be any, so how are we going to decide whether the failure was due to incompatibility?

Reply:
the command is to install a resource so from a user's point of view if a resource is getting installed then why warn, user is not trying to check compatibility. If it fails to install then the user has to know the reason why? then hub can show suggest that the hub was unable to check compatibility so please check the pipelines version and task version.